schema/transportation/segment.yaml

```diff
@@ -61,6 +61,26 @@ properties:
           "$comment": >-
             Each entry in this array is the GERS ID of a transportation connector feature.
         uniqueItems: true
+        minItems: 2
+        default: []
+      connectors:
```

Comment:
would be good to also update the description for `connector_ids` - that ids should be consumed from the `connectors` and a comment on deprecation